android.Dockerfile

```diff
@@ -0,0 +1,157 @@
+# Download Debian-9 (Stretch) with openjdk-8 that is required for Android.
```

Comment:
Wow... this is a long Dockerfile. Is there anything here we can inherit from other files, to void duplication?

Reply:
I have added a separate Dockerfile for Android only because Asylo's Dockerfile depends on Debian-10, that does not have Java-8 (only has Java-11). And Android requires Java-8 and does not support higher versions.